### PR TITLE
Demo project test

### PR DESCRIPTION
**Pull request by kblok**
This is more a public proof that the demo project works.

This guy won't build in every PR, only in master.

### CODE REVIEW

**Comment by kblok** [comment]
@Meir017

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 [![NuGet](https://img.shields.io/nuget/v/PuppeteerSharp.svg?style=flat-square&label=nuget&colorB=green)][NugetUrl]
 [![Build status](https://ci.appveyor.com/api/projects/status/pwfkjb0c4jfdo7lc/branch/master?svg=true)][BuildUrl]
+[![Build status](https://ci.appveyor.com/api/projects/status/10g64a4aa0083wgf/branch/master?svg=true)][BuildUrl]
```
Comment (Meir017) [inline README.md]: what is this?

File: demos/PuppeteerSharpPdfDemo/Program.cs [hunk 2]
```diff
@@ -10,10 +10,10 @@ class MainClass
     {
         public static void Main(string[] args)
         {
-            MainAsync().GetAwaiter().GetResult();
+            MainAsync(args).GetAwaiter().GetResult();
```
Comment (Meir017) [inline demos/PuppeteerSharpPdfDemo/Program.cs]: we could change the `Main` to be `async`

File: README.md (hunk 1 above)
Comment (kblok) [reply README.md]: The AppVeyor badge of the new project

**Pull request merged**